server/src/model/modification/modificationSchema.ts

```
@@ -65,15 +65,27 @@ export const modificationSchema = gql`
     currentPhaseName: PhaseName
   }
 
+  type CreateAmendmentResponse {
+    success: Boolean!
+    message: String
+  }
+
+  type CreateExtensionResponse {
+    success: Boolean!
+    message: String
+  }
```

I do like these simplified responses as what we're primarily interested in is just whether the operation succeeded (we generally just close the modal and refetch the queries so we hardly need any data back) and this means we can just add fields to this response type as they're needed.

That being said, it would be a good idea to move this into throwing an error proper rather than returning success. I generally like this level of control especially if GQL is going to return everything as a `200` anyway (removing the most canonical signal to rely on) but that's just some food for thought. Might be worth making sure we're actually throwing errors here as well